x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -10,18 +10,31 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const pageObjects = getPageObjects(['common', 'apiKeys']);
   const log = getService('log');
+  const security = getService('security');
 
   describe('Home page', function() {
     this.tags('smoke');
     before(async () => {
+      await security.testUser.setRoles(['kibana_admin']);
       await pageObjects.common.navigateToApp('apiKeys');
     });
 
+    after(async () => {
+      await security.testUser.restoreDefaults();
+    });
+
+    // https://www.elastic.co/guide/en/kibana/7.6/api-keys.html#api-keys-security-privileges
+    it('Shows required privileges ', async () => {
+      log.debug('Checking for required privileges method section header');
+      const message = await pageObjects.apiKeys.apiKeysPermissionDeniedMessage();
+      expect(message).to.be('You need permission to manage API keys');
+    });
+
     it('Loads the app', async () => {
+      await security.testUser.setRoles(['kibana_admin', 'test_api_keys']);
```

Comment:
Based on the comment in the original PR, can we try using Kibana feature controls in the `test_api_keys` role instead of adding the `kibana_admin` role?

Reply:
yep, zoomed with Larry to figure out feature controls in test_api_keys role. Modified the role.